### ISSUE TITLE
pylint not reporting missing import

### ISSUE THREAD

**Issue by asincero** [PyCQA/pylint#2879]
### Steps to reproduce
Directory structure:
* testapp/
    * earth/
        * bart.py
    * venus/
        * lisa.py
        * marge.py

In "earth/bart.py":
```python
# earth/bart.py
# pylint: disable=missing-docstring

import venus.lisa

def fee() -> None:
    venus.marge.fih()
```

In "venus/lisa.py":
```python
# venus/lisa.py
def foh() -> None:
    pass
```

In "venus/marge.py":
```python
# venus/marge.py
def fih() -> None:
    pass
```

Run:

pylint earth/bart.py

### Current behavior

No errors are reported.

### Expected behavior

An error on line 7 saying something to the effect of "venus.marge.fih()" is undefined because "venus.marge" was not imported.

### pylint --version output

pylint 2.1.1
asxtroid 2.0.4
Python 3.7.0 (default, Sep 20 2018, 14:56:27)
[GCC 6.3.0 20170516]

### Additional Comments
Importing "earth.bart" in a Python REPL session, then invoking "earth.bart.fee()" results in an error as expected.  Therefore, I feel pylint should be able to detect this.

If I change line 7 in "earth/bart.py" from "venus.marge.fih()" to "venus.marge.fih2()", pylint will output an error saying that "venus.marge" has no "fih2" member and will even suggest that I meant "fih" instead.

**Comment by PCManticore** [comment]
Thanks for opening an issue! I agree that this could be improved.
